scripts/release.ts

```diff
@@ -86,11 +109,32 @@ async function bumpVersions() {
     console.log(c.yellow('No version changes made. Exiting...'));
     process.exit(1);
   }
-  console.log(c.green('Bumped package versions.'));
+  console.log(c.green('Bumped package versions. Bumping dependencies next...'));
+
+  getPackages().forEach((pack) => {
+    const [packageName] = Object.keys(pack);
+    const [packageLocation] = Object.values(pack);
+    const jsonLocation = path.join(
+      root,
+      packageLocation.split('file:')[1].replaceAll('../', ''),
+      'package.json'
+    );
+    const json = require(jsonLocation);
+    const updatedJson = updateDSVersion(json);
+    writeJson(jsonLocation, updatedJson);
+    console.log(
+      c.green(`Bumped ${packageName} to @cmsgov/design-system@${newDesignSystemVersion()}.`)
+    );
+  });
 
   // Update versions.json
   const currentVersionsByPackage = updateVersions();
+  // Clean up our staged files
+  sh('git reset ./packages/docs/package.json');
+  sh('git checkout ./packages/docs/package.json');
```

Comment:
I'm not following why this is needed. Is there an opportunity for the comment above to explain why?

Reply:
Yeah I can add another comment. The goal is to not track/update the docs package.json, as happened before with Lerna. So we're explicitly removing the changes related to the docs site.

Reply:
Oh like you don't want to increment the version for the doc site?

Reply:
Did you mean for this to be on the tokens package?

Reply:
Tokens is handled on this [line](https://github.com/CMSgov/design-system/pull/3205/files#diff-21f9896491d70e481e031a1c66e9ffed07592f08455bdcd4a3cecc9c63814768R140).

We want to exclude both the docs and the tokens packages, which I think was the way things were working with the original Lerna-based script.

I'm not sure why it happens for docs and not tokens, but bumping the dependencies for the child system edits the package.json and stages it, just for docs though, so we need another explicit removal later in the script.

Reply:
Let's chat about this one all together live